src/applications/static-pages/sass/modules/_m-checklist-layout.scss

```diff
@@ -0,0 +1,7 @@
+.va-l-detail-page--checklist {
+  @media print {
```

Comment:
I know we don't use Tailwind CSS, but this conversation has sent me down a rabbit hole to see how other CSS utility libraries handle `print`. 

TIL https://tailwindcss.com/docs/breakpoints#styling-for-print